applications/matter_bridge/doc/matter_bridge_description.rst

```diff
@@ -532,18 +532,77 @@ Since the Matter stack uses one Bluetooth LE connection for commissioning, the m
 
 Increasing the number of Bluetooth LE connections affects the RAM usage on both the application and network cores.
 The current maximum number of Bluetooth LE connections that can be selected using the default configuration is ``10``.
-You can increase the number of Bluetooth LE connections if you decrease the size of the Bluetooth LE TX/RX buffers used by the Bluetooth controller, but this will decrease the communication throughput.
-Build the target using the following command in the project directory to enable a configuration that increases the number of Bluetooth LE connections to ``20`` by decreasing the sizes of Bluetooth LE TX/RX buffers:
 
-.. parsed-literal::
-   :class: highlight
+.. tabs::
 
-   west build -b nrf7002dk/nrf5340/cpuapp -- -DCONFIG_BRIDGED_DEVICE_BT=y -DEXTRA_CONF_FILE="overlay-bt_max_connections_app.conf" -Dipc_radio_EXTRA_CONF_FILE="overlay-bt_max_connections_net.conf"
+   You can configure the number of Bluetooth LE bridged devices depending on your device and configuration:
 
-.. parsed-literal::
-   :class: highlight
+   .. group-tab:: nRF53 DKs
+
+      The nRF53 Series supports the Matter bridge over Wi-Fi and Matter bridge over Thread configurations.
+
+      .. tabs::
+
+         .. group-tab:: Matter bridge over Wi-Fi
+
+            You can increase the number of Bluetooth LE connections if you decrease the size of the Bluetooth LE TX/RX buffers used by the Bluetooth controller, but this will decrease the communication throughput.
+
+            Build the target using the following command in the project directory to enable a configuration that increases the number of Bluetooth LE connections to ``20`` by decreasing the size of Bluetooth LE TX/RX buffers:
+
+            .. parsed-literal::
+               :class: highlight
+
+               west build -b nrf5340dk/nrf5340/cpuapp -p -- -Dmatter_bridge_SHIELD=nrf7002ek -DSB_CONFIG_WIFI_PATCHES_EXT_FLASH_STORE=y -DSB_CONFIG_DFU_MULTI_IMAGE_PACKAGE_WIFI_FW_PATCH=y -DSB_CONFIG_WIFI_NRF700X=y -Dmcuboot_CONFIG_UPDATEABLE_IMAGE_NUMBER=3 -DCONFIG_BRIDGED_DEVICE_BT=y -DEXTRA_CONF_FILE="overlay-bt_max_connections_app.conf" -Dipc_radio_EXTRA_CONF_FILE="overlay-bt_max_connections_net.conf" -DFILE_SUFFIX=nrf70ek
+
+         .. group-tab:: Matter bridge over Thread
+
+            You can not increase the default number of Bluetooth LE connections in this configuration.
```

Comment:
This paragraph is not quite true. You can't increase the number of supported Bluetooth LE connections using an overlay. But you can always do that by modifying the board files and decreasing the buffer sizes. I would also add a statement here that the default number of supported devices here is 9.

Reply:
Wording out Kamil's suggestion:
```suggestion
            You can not increase the default number of Bluetooth LE connections in this configuration using overlays.
```
And the following after Line 560:
```
            You can still increase the number of connections by modifying the board files and decreasing the buffer sizes.
            The default number of connections is nine.
```